playwright-e2e/tests/dsm/participant/participant-sending-sample-received-creates-germline-consent-addendum.spec.ts

```diff
@@ -65,11 +65,12 @@ test.describe.serial('Sending SAMPLE_RECEIVED event to DSS', () => {
       //Fill out an onc history and get back an accession number
       oncHistoryTab = await participantPage.clickTab<OncHistoryTab>(Tab.ONC_HISTORY);
       const oncHistoryTable = oncHistoryTab.table;
-      today = getDate();
-      randomAccessionNumber = await fillOncHistoryRow(participantPage, oncHistoryTab, facilityName, facilityPhoneNumber, facilityFaxNumber);
+      const lastRow = await oncHistoryTable.getRowsCount() - 1;
```

Comment:
The row index should be specified because selected PT can have existing rows in Onc History table.